### PR TITLE
Bugfix fileclass filegroup schema + more tests

### PR DESCRIPTION
**Pull request by vinjana**
Should be quickly reviewed.

### CODE REVIEW

File: RoddyCore/test/de/dkfz/roddy/tools/ComplexLineTest.groovy [hunk 1]
```diff
@@ -93,23 +91,48 @@ class ComplexLineTest {
     public ExpectedException thrown = ExpectedException.none()
 
     @Test
-    void testFaultyLine() {
+    void testMissingClosingParen() {
```
Comment (dankwart-de) [inline RoddyCore/test/de/dkfz/roddy/tools/ComplexLineTest.groovy]: Typo

Comment (vinjana) [inline RoddyCore/test/de/dkfz/roddy/tools/ComplexLineTest.groovy]: Wel, "paren" is the short form of "parenthesis" `)`

**Comment by vinjana** [comment]
Is the o.k. now? Can I merge?

**Comment by dankwart-de** [comment]
I thought, I merged it yesterday... ? Seems it did not happen. Yes, you're fine to go.

**Comment by dankwart-de** [comment]
Just did it.

**Pull request merged**